lib/sinatra.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
This isn't needed for the one string allocated here "sinatra/main" is an acceptable allocation that I doubt will be repeated enough to affect performance